src/balatro_gym/cards/interfaces.py

```diff
@@ -363,7 +371,7 @@ def deal(self, num: int) -> Sequence[PlayingCard]:
     def destroy(self, cards: Sequence[PlayingCard]) -> None:
         """Destroyed cards are removed permanently. Though, this can only happen to cards in-play."""
         for card in cards:
-            self._cards_remaining.remove(card)
```

Comment:
Why remove this line?

Reply:
Cards remaining lists all cards in the deck minus the cards that have been played (primarily happens in the `deal()` function. The union of both `deque`s give the full set of the `Deck`.